src/addressprovider_akonadi.cpp

```diff
@@ -73,7 +75,11 @@ bool AddressProviderPrivate::init()
 {
     _akonadiUp = false;
 #ifdef HAVE_AKONADI
-    if ( !Akonadi::Control::start( ) ) {
+    if ( Akonadi::ServerManager::state() == Akonadi::ServerManager::Broken ) {
```

Comment:
Can't we surround that with a Akonadi version check, to still keep it building on older akonadi versions that do not have the Broken state? Or is that already an old state?

Reply:
I don't know how old it is, but it's not new.